paddle/scripts/docker/Dockerfile.m4

```diff
@@ -1,12 +1,36 @@
 FROM PADDLE_BASE_IMAGE
 MAINTAINER PaddlePaddle Dev Team <paddle-dev@baidu.com>
-COPY build.sh /root/
-ENV GIT_CHECKOUT=v0.9.0a0
+
+# It is good to run apt-get install with Dockerfile RUN directive,
+# because if the following invocation to /root/build.sh fails, `docker
+# build` wouldn't have to re-install packages after we fix
+# /root/build.sh.
```

Comment:
Maybe add reference for docker build cache here? `https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/build-cache` (so for the curious ones they can jump to documentation right away)